test/Polly.Specs/Caching/CacheAsyncSpecs.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Polly.Specs.Caching;
+﻿using static Polly.Specs.DictionaryHelpers;
```

Comment:
If you made this a static global using for this project, we wouldn't need to add this to all these files:

```xml
<ItemGroup>
  <Using Include="Polly.Specs.DictionaryHelpers" Static="true" />
</ItemGroup>
```

Reply:
Done